### PR TITLE
add new params in constant.ts, fixes #828, #854

### LINKED ISSUE #828: move max number of transactions to constant.ts

### PR DESCRIPTION
**Pull request by domakun**

### CODE REVIEW

File: src/config/constants.ts [hunk 1]
```diff
@@ -142,6 +142,7 @@ export const networkConfig = {
   EMPTY_PUBLIC_KEY: '0000000000000000000000000000000000000000000000000000000000000000',
   PUBLIC_KEY_LENGTH: 64,
   NAMESPACE_MAX_LENGTH: 64,
+  maxNumberOfTransactions:100,
```
Comment (decentraliser) [inline src/config/constants.ts]: please put it in app params
`networkConfig `should only be used for values set server sides like in the file `config-network.properties`

Comment (domakun) [reply src/config/constants.ts]: maxNumberOfTransactions was set server sides

Comment (decentraliser) [reply src/config/constants.ts]: ah nice ok

**Comment by decentraliser** [comment]
let's fix the default hash lock issue in this one
https://github.com/nemfoundation/nem2-desktop-wallet/issues/854

**Pull request merged**